aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/Text.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.api.annotations.widgets;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.exadel.aem.toolkit.api.annotations.meta.PropertyMapping;
+import com.exadel.aem.toolkit.api.annotations.meta.ResourceType;
+import com.exadel.aem.toolkit.api.annotations.meta.ResourceTypes;
+
+/**
+ * Used to set up
+ * https://helpx.adobe.com/experience-manager/6-3/sites/developing/using/reference-materials/granite-ui/api/jcr_root/libs/granite/ui/components/coral/foundation/text/index.html
+ * A generic text component that is rendered as <span> in TouchUI dialog
+ */
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@ResourceType(ResourceTypes.TEXT)
+@PropertyMapping
+@SuppressWarnings("unused")
+public @interface Text {
+    /**
+     * Maps to the text of the component of this TouchUI dialog component's node
+     * @return String value
+     */
+    String text();
```

Comment:
Also, an idea (discussable) maybe we can rename this property to `value()` (while still mapping it to `text` JCR property, of course?
It will allow the users to use it in a more simple manner `@Text("Cats!")` instead of `@Text(text="Cats!")`
CC: @smiakchilo , @Alexey-Chernyakov